alex_px-moscow.md

```diff
@@ -3,5 +3,7 @@
 ### Restaurants
 
 ### Local Attractions
+- Red Square
+- Historical Museum
 
 ### Remote Attractions
```

Comment:
@shinton 
Nothing special this month :disappointed: 
Next national holiday is on 4th of November.